roles/ceph-ansible/defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# home directory which contains copy of files in ceph-ansible package
+home_dir: "{{ ansible_env['WORKSPACE'] | default(ansible_env['HOME']) }}"
+
+# list of devices which will be used for Ceph.
+# If it is empty, list is autodetected by ceph-ansible
+devices: []
+
+# use collocated journals? Default to true for simplicity.
+collocation: true
+
+# if collocation is false and journal devices are set,
+# create journals on journal devices.
+# Default is empty because collocation is true.
+# collocation == false and len(journal_devices) == 0 is not supported!
+journal_devices: []
+
+# path to ansible configuration which overrides default ansible configuration
+ansible_conf: /tmp/ansible.cfg
```

Comment:
I think that is not problem, because it is written every time you run this playbook. This is not configuration file which should persist after reboot.